### PR TITLE
✨ inherit free usage from affiliate

### PR DESCRIPTION
**Pull request by m0ppers**
fixes https://github.com/railslove/recover-backlog/issues/91

I don't get why there is this menu_alias change but I guess it doesn't hurt?

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -104,8 +105,8 @@
     t.datetime "block_at"
     t.bigint "frontend_id"
     t.string "api_token"
-    t.string "menu_alias"
     t.integer "auto_checkout_minutes"
+    t.string "menu_alias"
```
Comment (salimhb) [inline db/schema.rb:109]: maybe it doesn't hurt to merge the change, but it also doesn't hurt to not do it ;)

**Review by salimhb — CHANGES_REQUESTED** [review]
This doesn't look like it covers all the points from https://github.com/railslove/recover-backlog/issues/91

>  Then I can use recover for free for a given time

Instead of setting `can_use_for_free`, I suggest simply extending the trial period to X days, where X is a property of Affiliate called `trial_days`

File: db/schema.rb (hunk 1 above)
Comment (m0ppers) [reply db/schema.rb]: removed it

**Comment by m0ppers** [comment]
ticket does the wrong thing just as salim noted :1st_place_medal: 
doing a new PR

**Pull request closed without merging**